### DISCUSSION TITLE
Refresh file browser button

### DISCUSSION THREAD

**Discussion by tritrek** [metabrainz/picard-docs#175] [Ideas]
Sometimes, when directory/file content changes it's not reflected in Picard. E.g., a new folder is created or new files appear in a folder, however, Picard doesn't 'see' them, the only solution at the moment is to quit and restart Picard, which is not optimal. I think there's a need for a button or function to refresh file browser content.

**Comment by phw** [comment]
Please report bug reports or feature requests in Picard's issue tracker at https://tickets.metabrainz.org/projects/PICARD/issues or discuss usage questions in the community forums at https://community.metabrainz.org/c/picard/ . This repository here is solely about the documentation projects. 

For your actual issue please see PICARD-2108
